### ISSUE TITLE
Accessibility: VoiceOver does not navigate in Images and Videos view

### ISSUE THREAD

**Issue by OzancanKaratas** [deltachat/deltachat-ios#1580]
- Device: iPhone SE (1st generation)
- iOS version: 15.4.1
- Delta Chat version: Latest beta on TestFlight
- Expected behavior: VoiceOver should read the Images and Videos view
- Actual behavior: VoiceOver reads only the page title
- Steps to reproduce the problem:
  1. Open a profile where any video or image was shared
  2. Select Images and Videos
- Screenshots: No screenshots as this issue is related to UIAccessibility
- Logs: I couldn't get a log

Note: I haven't been able to test the Files view.

**Comment by cyBerta** [comment]
What information should be read out for images and videos? I think we don't have much meta-data, besides that I just tried Signal and Telegram, both don't show images and videos (or even Documents!) with VoiceOver. 

The iOS photos app reads out: "Photo, $Date" or "Video, $Date". 

We could theoretically add the sender information for example. We could also add to the accessibility hint the message text if there's any. 
In that case it could be sth. like: "Photo, 12. May, sent by cyberta - (pause) - Hi there, look where I've been yesterday."

**Comment by cyBerta** [comment]
Ok, I just learned about the newish VoiceOver image recognition features and tried them out with the photos app. It is quite impressive. It would be really cool if we can support that. The best is the photo recognition works - contrary to Android - locally on the phone.

**Comment by OzancanKaratas** [comment]
I'd like to give some information:

Images and Photos view is not treated as a list, but Files view is.

Please do not use Telegram to accessibility tests. Because it has known accessibility issues.
